R/configuration.R

```diff
@@ -270,6 +277,10 @@ VCRConfig <- R6::R6Class(
     write_disk_path = function(value) {
       if (missing(value)) return(private$.write_disk_path)
       private$.write_disk_path <- value
+    },
+    verbose_errors = function(value) {
+      if (missing(value)) return(private$.verbose_errors)
+      private$.verbose_errors <- value
```

Comment:
Maybe add an `is.logical()` assertion here.

Reply:
will do